modules/handlers/anonym_comment.py

```diff
@@ -16,5 +20,42 @@ def anonymous_comment_msg(update: Update, context: CallbackContext):
     info = EventInfo.from_message(update, context)
 
     if info.chat_id == config_map['meme']['channel_group_id'] and info.message.via_bot is None:
-        info.message.copy(chat_id=info.chat_id, reply_to_message_id=info.message.reply_to_message.message_id)
+        message = info.message
+        poll = message.poll  # if the message is a poll, get its reference
+        anon_credits = 'by: ' + choice(read_md("anonym_names").split("\n"))
+        try:
+            if poll or message.sticker:  # makes sure the poll is anonym
+              if poll:
+                sent_message_id = info.bot.send_poll(chat_id=info.chat_id,
+                                                  question=f'{poll.question}\n\n{anon_credits}',
+                                                  options=[option.text for option in poll.options],
+                                                  type=poll.type,
+                                                  allows_multiple_answers=poll.allows_multiple_answers,
+                                                  reply_to_message_id=info.message.reply_to_message.message_id if info.message.reply_to_message else None,
+                                                  correct_option_id=poll.correct_option_id).message_id
+              else:
+                sent_message_id = info.bot.send_sticker(chat_id=info.chat_id,
+                                                      sticker=message.sticker.file_id,
+                                                      reply_to_message_id=info.message.reply_to_message.message_id if info.message.reply_to_message else None).message_id
+
+              info.bot.send_message(chat_id=info.chat_id,
```

Comment:
No need to send it for the poll, since you adjust the question

Reply:
the poll question has limited amount of chars, so it can be a problem

Reply:
Then you should change
```diff
- question=f'{poll.question}\n\n{anon_credits}'
+ question=poll.question
```
and leave the if statements as you made them. Mind the indentation though, the spaces are not always 4

Reply:
oh i forgot to remove this :man_facepalming: